src/pocketmine/Player.php

```diff
@@ -2266,7 +2266,7 @@ public function handleDataPacket(DataPacket $packet){
 					break;
 				}
 
-				if(!in_array($packet->protocol, ProtocolInfo::ACCEPTED_PROTOCOLS)){
+				if($packet->protocol !== ProtocolInfo::CURRENT_PROTOCOL){
```

Comment:
really? what if there's another one of those updates that changes the protocol number but not the protocol, like 0.15.9 to 0.15.10? all that did was add a texture pack.